### DISCUSSION TITLE
Gallery Overview

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#162] [General]
# Gallery Overview

The gallery module provides a simple layout for images.

https://hbstack.dev/en/docs/modules/gallery/overview/

**Comment by cbeng** [comment]
(UPDATED/RESOLVED below) 

I'm encountering an issue where some, but not all, of my portrait orientation images are presented in landscape orientation rotated to the left in the gallery. Any ideas how to solve it?

UPDATE: After some research, I see these issues can arise with Hugo's Resize function related to image exif data. I fixed the issue by first running my images through a minifier (e.g. ImageOptim or PNGQuant) which strips exif data.

**Comment by razonyang** [comment]
Hi @cbeng, the bigger picture module now (since `v0.9.0`) supports rotating, flipping and playing images, may help visitors to handle those images.

```sh
hugo mod get -u github.com/hbstack/bigger-picture
```

![image](https://user-images.githubusercontent.com/17720932/235419369-4b0a3298-c334-40e1-88fa-a3034d5cb3e2.png)

**Comment by LITUATUI** [comment]
Can you add support for AVIF image format?

**Reply by razonyang** [reply]
No, since Hugo doesn't support (processing) avif images, we require images methods to resize, get sizes, get dates and so on.
